protocols/raft/src/main/java/io/atomix/protocols/raft/metrics/RaftRoleMetrics.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016-present Open Networking Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.atomix.protocols.raft.metrics;
+
+import io.prometheus.client.Counter;
+import io.prometheus.client.Gauge;
+import io.prometheus.client.Histogram;
+
+public class RaftRoleMetrics extends RaftMetrics {
+
+  private static final Gauge ROLE =
+      Gauge.build()
+          .namespace("atomix")
+          .name("role")
+          .help("Shows current role")
+          .labelNames("partition")
+          .register();
+
+  private static final Counter HEARTBEAT_MISS =
+      Counter.build()
+          .namespace("atomix")
+          .name("heartbeat_miss_count")
+          .help("Count of missing heartbeats")
+          .labelNames("last", "partition")
```

Comment:
I think it will map the partition value to the "last" label if this is still mentioned, no?

Reply:
stupid me i try to do too many things concurrently